openedx/features/edly/tests/factories.py

```diff
@@ -51,3 +53,43 @@ class Meta(object):
     edx_organization = SubFactory(OrganizationFactory)
     lms_site = SubFactory(SiteFactory)
     studio_site = SubFactory(SiteFactory)
+
+
+class EdlyUserProfileFactory(DjangoModelFactory):
+    """
+    Factory class for "EdlyUserProfile" model.
+    """
+    class Meta(object):
+        model = EdlyUserProfile
+        django_get_or_create = ('user', )
+
+    id = Sequence(u'{0}'.format)
+    user = None
+
+    @factory.post_generation
+    def edly_sub_organizations(self, create, extracted, **kwargs):
+        if not create:
+            return
+
+        if extracted:
+            for edly_sub_org in extracted:
+                self.edly_sub_organizations.add(edly_sub_org)
+
+
+class EdlyUserFactory(UserFactory):
```

Comment:
we don't need this factory only `EdlyUserProfileFactory` should be enough.

Reply:
@ziafazal We added `EdlyUserFactory` to verify that when a user is created its related `EdlyUserProfile` object is automatically created.